docs/site.md

```diff
@@ -61,21 +61,21 @@ On top of the defaults of Laika standalone, sbt-typelevel-site adds:
 
 #### Additional Defaults for Typelevel Projects
 
-If the generated documentation is for a Typelevel project, you can optionally enable a set of additional defaults
-on top of the generic defaults listed in the previous section:
-
-```scala
-tlSiteIsTypelevelProject := true
-```
```

Comment:
Wouldn't it be helpful to keep an updated version of this? What I mean is, it seems far more likely an affiliate project would like to know how they can opt-in to the Typelevel theming, rather than an org project wanting to know how it can opt-out.

Reply:
Yes, I just wasn't sure about the intention here. The `Organization` value is package-private and the `Affiliate` is not, yet in the implementation there is no difference between the two. I wasn't exactly sure what these design choices meant, so I did not write about it. 🙂

Reply:
I added a section for `Affiliate` projects based on my somewhat limited understanding of the intentions. Feel free to rephrase or expand if necessary.

Reply:
> yet in the implementation there is no difference between the two.

We decided in https://github.com/typelevel/governance/issues/34#issuecomment-1256693650 to start collecting this information even though we don't know what do with it yet, if anything.

> The `Organization` value is package-private and the `Affiliate` is not,

Yes, the idea is to discourage users from claiming they are an organization project. We'll see how it works 😀